hosting/docker.md

```diff
@@ -0,0 +1,57 @@
+---
+icon: container
+order: 90
+---
+
+![](/static/headers/hosting_github-pages.png)
```

Comment:
Maybe another header would be nice, but im not an artist, so i just re-used the github-pages one